scripts/update-se-drivers.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const cheerio = require('cheerio');
+const got = require('got');
+const chromeDriverDownloadUrl = 'https://chromedriver.chromium.org/downloads';
+const { execSync } = require('child_process');
+
+let chromeVersion;
+let chromeDriverVersion;
+let foundDriverVersion = false;
+
+function getOS() {
+  var os = process.platform;
+  if (os == "darwin") {
+    os = "MacOS";
+  } else if (os == "win32" || os == "win64") {
+    os = "Windows";
+  } else if (os == "linux") {
+    os = "Linux";
+  }
+  return os;
+}
+
+got(chromeDriverDownloadUrl).then(async(response) => {
+  os = getOS();
+  console.log(`Operating System - ${os}`);
+
+  if (os === 'MacOS') {
+    chromeVersionString = execSync(`/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --version`) + '';
+    // Example output of chromeVersionString - Google Chrome 83.0.4103.97
+    // Split this string by spaces to get the version number
+    chromeVersion = chromeVersionString.split(" ")[2];
+  } else {
+    chromeVersion = execSync(`google-chrome --product-version`) + '';
+  }
+
+  console.log(`Chrome Version - ${chromeVersion}`);
+
+  const $ = cheerio.load(response.body);
+
+  // Chromedriver download URLs are of the form https://chromedriver.storage.googleapis.com/index.html?path=83.0.4103.39/
+  // Filter through all href of this format, depending on the chrome version installed and get the chrome driver version from that
+  $('a').filter(isChromeDriverDownloadUrl).each((i, link) => {
```

Comment:
how do you think if we do
- extra major chrome version
- fetch for `base-url/LATEST_RELEASE_83` to extra chrome version
- fetch for  `base-url/chrome-version-from-step-2`

Reply:
I'm not sure I understand. 
I'm trying to get the chromedriver version corresponding to the chrome binary version.
We can sync up to discuss this on slack.

Reply:
the way you're implementing now is 
- search through page https://chromedriver.chromium.org/downloads to find chromedriver version matched major chrome version.

what I am saying, given chrome major version, we could directly
- fetch file like : `https://chromedriver.storage.googleapis.com/LATEST_RELEASE_83` (83 is a major chrome version)
- the response of that url is just the chrome driver version.

Reply:
Now, I get it. Thanks!
This makes the script much more cleaner and simpler.
Have updated it. Take a look.